packages/notifications/tests/dummy/config/ember-cli-update.json

```diff
@@ -0,0 +1,20 @@
+{
+  "schemaVersion": "1.0.0",
```

Comment:
Should this be deleted?

Reply:
No, we want to keep this. We have this in other projects as well.

>it's a file to help ember-cli-update be more efficient and correct with future upgrades
it also can be aware of different blueprints your project might be using

Reply:
Ah gotcha it was confusing seeing it under `tests/dummy/config/` and I couldn't remember it from existing projects